models/engine/db_storage.py

```diff
@@ -74,3 +74,13 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """ Retrieve one object """
+        obj_key = eval(cls).__name__ + "." + id
+        return self.__objects.get(obj_key, None)
```

Comment:
please fix this error
AttributeError: 'DBStorage' object has no attribute '_DBStorage__objects'

""   test ""
>> city = storage.get('City', '531a55f4-7d82-47d9-b54c-a76916479556')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/luis/AirBnB_clone_v3/models/engine/db_storage.py", line 81, in get
    return self.__objects.get(obj_key, None)
AttributeError: 'DBStorage' object has no attribute '_DBStorage__objects'